src/test/java/org/mockbukkit/mockbukkit/block/state/DecoratedPotStateMockTest.java

```diff
@@ -116,13 +116,13 @@ void getSnapshot()
 	@Test
 	void notEqualNull()
 	{
-		assertNotEquals(pot, null);
+		assertNotEquals(null, pot);
 	}
 
 	@Test
 	void notEqualObject()
 	{
-		assertNotEquals(pot, new Object());
+		assertNotEquals(new Object(), pot);
```

Comment:
```suggestion
		// Checks if the pot is not equal to the object using the pot equals method, don't swap
		assertNotEquals(pot, new Object());
```
Might be a reduntant check though